### PR TITLE
Use differnt query and index to speedup history table

### PR DESCRIPTION
**Pull request by Jan-M**
Implement different query

### CODE REVIEW

File: database/eventlog/00_create_schema.sql [hunk 1]
```diff
@@ -12,8 +12,8 @@ CREATE TABLE zmon_eventlog.events(
   e_data jsonb
 );
 
-CREATE INDEX ON zmon_eventlog.events USING gin (e_data);
-CREATE INDEX ON zmon_eventlog.events (e_created , e_type_id);
+CREATE INDEX ON zmon_eventlog.events (((e_data->'checkId')::text), e_created);
+CREATE INDEX ON zmon_eventlog.events (((e_data->'alertId')::text), e_created);
```
Comment (lmineiro) [inline database/eventlog/00_create_schema.sql:17]: The queries also filter by the e_type_id, won't the removal of that one hurt?

Comment (Jan-M) [reply database/eventlog/00_create_schema.sql:17]: imho it does not matter that much, we query basically for all event types with alertId or checkId

**Comment by Jan-M** [comment]
:+1:

**Comment by lmineiro** [comment]
👍

**Pull request merged**